packages/css/src/components/Form/form.css

```diff
@@ -145,3 +145,99 @@ textarea.form-input + .form-icon {
   top: var(--form-input-large-padding-y);
   bottom: unset;
 }
+
+/* RADIO & CHECKBOX */
+.form-control {
+  display: flex;
+  align-items: center;
+  gap: var(--margin-1);
+}
+
+.form-control .form-label {
+  font: var(--form-check-radio-label);
+}
+
+.form-control input[type="radio"],
+.form-control input[type="checkbox"] {
+  margin: 0;
+  width: 22px;
+  height: 22px;
+  outline: none;
+  border: 1px solid var(--form-border-color);
+  background: var(--form-background-color);
+  font: inherit;
+  appearance: none;
+}
+
+.form-control input[type="radio"] {
+  border-radius: 50%;
+  transition: 120ms border ease-in-out;
+}
+
+.form-control input[type="radio"]:checked {
+  border-width: 7px;
+  border-color: var(--form-check-radio-default);
+}
+
+.form-control input[type="radio"]:checked:hover {
+  border-color: var(--form-check-radio-hover);
+}
+
+.form-control input[type="radio"]:checked:disabled {
+  border-color: var(--form-disabled-border-color);
+  background: var(--form-disabled-input-color);
+}
+
+.form-control input[type="radio"]:not(:checked).error,
+.form-control input[type="checkbox"]:not(:checked).error {
+  border-color: var(--form-error-border-color);
+  background: var(--form-background-color);
+}
+
+.form-control input[type="checkbox"] {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: var(--form-input-border-radius);
+  transition: 120ms background-color ease-in-out;
+}
+
+.form-control input[type="checkbox"]:checked {
+  border-color: var(--form-check-radio-default);
+  background: var(--form-check-radio-default);
+  font-size: 22px;
+}
+
+.form-control input[type="checkbox"]:checked::before {
+  color: var(--form-background-color);
+  content: "\f33e";
+  text-align: center;
+  text-decoration: inherit;
+  text-transform: none;
+  font: normal normal normal 1em/1 Phosphor;
```

Comment:
What if the Design system stop using Phosphor - font or font name should come from a variable